tests/polkadot_tests/libraries.rs

```diff
@@ -74,7 +74,7 @@ fn using() {
     let mut runtime = build_solidity(
         r##"
         contract test {
-            using ints for uint32;
```

Comment:
This potentially reduces test coverage, or is it tested in another place anyways?

Reply:
This is already tested above (line 50 to 70)